### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Dusch4593**
Make some small edits to the `CONTRIBUTING.md` guide including updating outdated information.

### CODE REVIEW

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@Dusch4593 After approving this, I went over the contributing markdown and noticed that beginners may not know what GitHub UI is referring to.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Contributing follows this workflow:
 
 If you haven't gone through this workflow before, you can check out [this GitHub tutorial](https://github.com/firstcontributions/first-contributions#readme) (highly recommend) or [this YouTube video](https://www.youtube.com/watch?v=rgbCcBNZcdQ) to learn about how to make a PR from a fork using Git.
 
-Alternatively, if you'd prefer to keep things to the GitHub UI, you can follow the instructions in that video up to 1:18 to fork this repo. After that, you can create your entry in your fork using the UI and then make a PR by pressing this handy button:<br>
+Alternatively, if you'd prefer to work with the GitHub UI, you can follow the instructions in that video up to 1:18 to fork this repo. After that, you can create your entry in your fork using the UI and then make a PR by pressing this handy button:<br>
```
Comment (Dusch4593) [inline .github/CONTRIBUTING.md:35]: @yangc95 What about something like this?

```suggestion
Alternatively, if you'd prefer to work directly on GitHub, you can follow the instructions in that video up to 1:18. After forking this repo, you can create your entry in your fork using the website's user interface and make a PR by pressing this handy button:<br>
```

**Comment by Dusch4593** [comment]
> @Dusch4593 After approving this, I went over the contributing markdown and noticed that beginners may not know what GitHub UI is referring to.

What do you think of [this possible edit](https://github.com/Codecademy/docs/pull/1020/files#r982647852)?

**Comment by Dusch4593** [comment]
> > @Dusch4593 After approving this, I went over the contributing markdown and noticed that beginners may not know what GitHub UI is referring to.
> 
> What do we think of [this possible edit](https://github.com/Codecademy/docs/pull/1020/files#r982647852)?

**Pull request merged**